docs/internals/contributing/bugs-and-features.txt

```diff
@@ -39,8 +39,8 @@ particular:
 * **Do** read the :doc:`FAQ </faq/index>` to see if your issue might
   be a well-known question.
 
-* **Do** ask on |django-users| or `#django`_ *first* if you're not sure if
-  what you're seeing is a bug.
+* **Do** ask on |django-users| or the `Django Discord server`_ *first* if
```

Comment:
Thought: If we're calling out other platforms, is it worth linking to the forum here too? There's a fair amount of "this looks weird" threads.

Reply:
I debated about it but I think this would be part of ticket-35908 (retiring the mailing lists) as in my head - Discord roughly replaces IRC and the forum roughly replaces the mailing lists

Reply:
That format separation makes sense to me :+1: I'll stick a comment on #18808